rosa/rosa-wrapper.py

```diff
@@ -105,6 +106,71 @@ def _install_addons(rosa_cmnd,cluster_id,addons):
+    machinepool_cmd = [rosa_cmnd, "create", "machinepool",
+                       "--cluster", cluster_id,
+                       "--instance-type", args.machinepool_flavour,
+                       "--name", args.machinepool_name,
+                       "--labels", args.machinepool_labels,
+                       "--taints", args.machinepool_taints,
+                       "--replicas", str(args.machinepool_replicas),
+                       "-y"]
+    logging.debug(machinepool_cmd)
+    machinepool_process = subprocess.Popen(machinepool_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    machinepool_stdout,machinepool_stderr = machinepool_process.communicate()
+    if machinepool_process.returncode != 0:
+        logging.error('Unable to create machinepool %s on %s' % (args.machinepool_name,cluster_id))
+        logging.error(machinepool_stdout.strip().decode("utf-8"))
+        logging.error(machinepool_stderr.strip().decode("utf-8"))
+    else:
+        if args.machinepool_wait:
+            logging.info('Created machinepool %s on %s. Waiting up to %d seconds for hosts to come up' % (args.machinepool_name,cluster_id,args.machinepool_wait_cycles*5))
+            logging.info('Checking if oc tool is available on the system')
+            oc_cmd = ["oc", "-h"]
+            logging.debug(oc_cmd)
+            oc_process = subprocess.Popen(oc_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            oc_stdout,oc_stderr = oc_process.communicate()
+            if oc_process.returncode != 0:
+                logging.error('%s unable to execute -h' % oc_cmd)
+                logging.error(oc_stdout.strip().decode("utf-8"))
+                logging.error(oc_stderr.strip().decode("utf-8"))
+            else:
+                kubeconfig_env = os.environ.copy()
+                kubeconfig_env["KUBECONFIG"] = kubeconfig
+                # 60 cicles, waiting 5 seconds at the end of each cicle, is about 300 seconds (5 minutes)
+                for counter in range(1,args.machinepool_wait_cycles):
+                    nodecheck_cmd = ["oc", "get", "nodes", "--no-headers=true",
+                                     "-l", args.machinepool_labels,
+                                     "-o", "custom-columns=NAME:metadata.name,STATUS:status.conditions[-1].type"]
+                    logging.debug(nodecheck_cmd)
+                    nodecheck_process = subprocess.Popen(nodecheck_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=kubeconfig_env)
+                    nodecheck_stdout,nodecheck_stderr = nodecheck_process.communicate()
+                    if nodecheck_process.returncode != 0:
+                        logging.error('Unable to oc get command, cannot check node status')
+                        logging.error(nodecheck_stdout.strip().decode("utf-8"))
+                        logging.error(nodecheck_stderr.strip().decode("utf-8"))
+                        break
```

Comment:
Here I see a break, maybe the break is needed above.

Reply:
fixed